include/zephyr/bluetooth/mesh/main.h

```diff
@@ -762,6 +762,73 @@ struct bt_mesh_friend_cb {
 	static const STRUCT_SECTION_ITERABLE(bt_mesh_friend_cb,          \
 					     _CONCAT(bt_mesh_friend_cb_, \
 						     _name))
+#if defined(CONFIG_BT_TESTING)
+struct bt_mesh_snb {
+	/** Flags */
+	uint8_t flags;
+
+	/** Network ID */
+	uint64_t net_id;
+
+	/** IV Index */
+	uint32_t iv_idx;
+
+	/** Authentication Value */
+	uint64_t auth_val;
+};
+
+#if defined(CONFIG_BT_MESH_V1d1)
+struct bt_mesh_prb {
+	/** Random */
+	uint8_t random[13];
```

Comment:
Why do you need 13 bytes if only 8 bytes are copied to the array?

Reply:
fixed